Samples/MiniFramework/VulkanApp.cs

```diff
@@ -73,6 +74,11 @@ public void Initialize(IVulkanAppHost host)
             // Create a command buffer for each swapchain image.
             CommandBuffers = Context.GraphicsCommandPool.AllocateBuffers(
                 new CommandBufferAllocateInfo(CommandBufferLevel.Primary, SwapchainImages.Length));
+            // Create a fence for each commandbuffer so that we can wait before using it again
+            _initializingPermanent = true; //We need our fences to be there permanently
```

Comment:
This '_initializingPermanent = true' is pretty ugly here but there is no reason that we need to recreate the fences on resize. Any suggestion? 😅

Reply:
Oh yeah, I think this was written quite some time ago and using `_initializingPermanent` to decide for disposable scope is indeed a bit weird 😁.